docs/source/endpoints/translations.md

```diff
@@ -128,3 +128,12 @@ This endpoint returns the proper placement for the newly created translation:
 This endpoint can be used with the {doc}`expansion` mechanism which allows getting additional information about a content item in one query, avoiding unnecessary requests.
 
 If a simple `GET` request is done on the content item, a new entry will be shown on the `@components` entry, with the URL of the `@translations` endpoint:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/translations_expand_get.req
```

Comment:
Here first a simple GET without expansion to show the 'translations' part ( with only '@id') unexpanded.
Use the  '/plone/en/test-document?expand=translations' without the '@' sign and no request body below.

Reply:
Would you please first provide a simple request and response for the unexpanded case.
Remember to remove the body of the GET request.

Reply:
Sorry I am unable to understand much, but what I get is that I have to make 2 get requests, one without the expansion to this `plone/en/test-document?@translations` and another get request to `plone/en/test-document?expand=translations` without  `@` sign. In both the request, I don't have to provide the body of get. 
right? @ksuess

Reply:
@Akshat2Jain HTTP examples of expansions are here:

- https://6.docs.plone.org/plone.restapi/docs/source/endpoints/navroot.html#expansion

...and docs of expansions:

- https://6.docs.plone.org/plone.restapi/docs/source/usage/expansion.html